Plots/Skew-T/NCL_skewt_3_2.py

```diff
@@ -0,0 +1,117 @@
+"""
+NCL_Skew_3_2.py
```

Comment:
There are technically two versions of this projection on the NCL website. skew_t_3_1 uses fahrenheit for the temperature (x) axis and doesn't thin the wind barbs. I can still make this change, but wanted to ensure it is known that a second version of this plot also exists.

Reply:
That makes sense. This is nitpicky but the file name in the leading comments is currently `NCL_Skew_3.py` but 'skew' shouldn't be capitalized and it's missing the 't'. It should be `NCL_skewt_3.py` or `NCL_skewt_3_2.py` if you decide to still differentiate between the Celsius and Fahrenheit examples. Whichever you pick, just make sure that it matches the actual filename.

We may want to have a conversation as a team about how we want to address examples that have multiple plots. We can divide them into separate example scripts or keep them together and choose the overarching example name. I think there have already been a few instances of considering this kind of choice for naming conventions.

Reply:
I don't mind using the way you suggested, it would make things a little bit easier when I update the NCL website with the links to the gallery page. There are a couple of examples on the gallery currently (I think its maybe under 5) that are using the separation of plots method that I have here so maybe renaming them will be a future issue/PR. The only benefit I could see for my method is if the author is only planning on doing one of the plots/examples and not all of them. But this could also maybe be addressed when they are linked (ex. "an example of plot_3_1 is available here") to prevent any confusion from users. Let me know what you think!

Reply:
Since MetPy doesn't have a built in way to change the temperature scale to Fahrenheit I think you can go ahead and name this file `NCL_skewt_3_2.py` since we won't be able to make the full two-plot example.

I will have to take a look at the examples with multiple plots that are already in the gallery, and then I'll open an issue to discuss this further with everyone. If we decided to separate the plots into different files and therefore different pages on the GeoCAT gallery, it wouldn't be hard to link "sister" examples together. Thanks for discussing options for how we should do this in the future.